packages/dataviews/src/stories/dataviews-picker.story.tsx

```diff
@@ -171,6 +173,7 @@ const DataViewsPickerContent = ( {
 				itemListLabel="Galactic Bodies"
 				defaultLayouts={ {
 					[ LAYOUT_PICKER_GRID ]: {},
+					[ LAYOUT_PICKER_TABLE ]: {},
```

Comment:
This PR's testing really nicely for me so far! This is such a minor point, but I was wondering if it's worth defaulting just the table view in this story to show more of the fields by default? Just to get a slightly more real world sense of how this view would look, because otherwise these stories only show the title and media fields. I.e. defaulting it to look more like this:

<img width="1276" height="885" alt="image" src="https://github.com/user-attachments/assets/fe244291-9f15-457c-8958-8385e8cf27b7" />

Feel free to skip this idea, though, we can always keep tweaking the storybook stories for these in follow-ups.